Controller/ThreadController.php

```diff
@@ -57,6 +57,12 @@ public function cgetAction(Request $request)
         $fieldDescriptors = $this->getFieldDescriptors();
         $restHelper->initializeListBuilder($listBuilder, $fieldDescriptors);
 
+        foreach ($request->query->all() as $filterKey => $filterValue) {
```

Comment:
maybe add a test for that? if we change the implementation we can check if it works the same.